salt/lax/bot-lax-adaptor.sls

```diff
@@ -194,3 +197,11 @@ uwsgi-bot-lax-adaptor:
 #        - require:
 #            - uwsgi-bot-lax-adaptor
 
+
+periodically-remove-expired-cache-entries:
+    cron.present:
+        - user: {{ pillar.elife.deploy_user.username }}
+        - name: cd /opt/bot-lax-adaptor/ && ./clear-expired-requests-cache.sh
+        - identifier: rm-expired-cache-entries
+        - special: "@weekly"
```

Comment:
could we pin a particular day?

Reply:
IIRC this would be only on `lax--*--1` and not on node 2 of an environment, would cross check that

Reply:
> could we pin a particular day?

Of course we can pin on a particular day. Which day would you like? I'll go with Monday ;)

Reply:
fyi, encountered this bug switching away from "@special": https://github.com/saltstack/salt/issues/44530